code/modules/client/loadout/loadout_suit.dm

```diff
@@ -79,6 +79,47 @@
 	path = /obj/item/clothing/suit/hooded/wintercoat/miner
 	allowed_roles = list(JOB_NAME_SHAFTMINER)
 
+//NOSTALGIC WINTER COATS
+
+/datum/gear/suit/oldwintercoat
+	subtype_path = /datum/gear/suit/oldwintercoat
+	cost = 6000
+
+/datum/gear/suit/oldwintercoat/grey
+	display_name = "nostalgic winter coat"
+	path = /obj/item/clothing/suit/hooded/wintercoat/old
+	cost = 3000
+
+/datum/gear/suit/oldwintercoat/security
+	display_name = "nostalgic security winter coat"
+	path = /obj/item/clothing/suit/hooded/wintercoat/security/old
+	allowed_roles = list(JOB_NAME_SECURITYOFFICER, JOB_NAME_BRIGPHYSICIAN, JOB_NAME_HEADOFSECURITY)
+
+/datum/gear/suit/oldwintercoat/medical
+	display_name = "nostalgic medical winter coat"
+	path = /obj/item/clothing/suit/hooded/wintercoat/medical/old
+	allowed_roles = list(JOB_NAME_PARAMEDIC, JOB_NAME_MEDICALDOCTOR, JOB_NAME_CHIEFMEDICALOFFICER, JOB_NAME_CHEMIST, JOB_NAME_GENETICIST)
```

Comment:
Brig physician should be allowed medical coat

Reply:
I just copypasted roles from the original winter coats, updating them both now. Should Virologist get access to medical winter coats too?

Reply:
Yea sure